static/src/javascripts/projects/commercial/modules/check-dispatcher.js

```diff
@@ -68,39 +56,6 @@ const checksToDispatch = {
             lowPriorityAdLoaded => !lowPriorityAdLoaded
         );
     },
-
-    isOutbrainBlockedByAds(): Promise<boolean> {
-        const dependentChecks = [
-            waitForCheck('hasHighPriorityAdLoaded'),
-            waitForCheck('hasLowPriorityAdLoaded'),
-        ];
-
-        return Promise.all(dependentChecks).then(results =>
-            everyCheckPassed(results)
-        );
-    },
-
-    isOutbrainMerchandiseCompliant(): Promise<boolean> {
-        const dependentChecks = [
-            waitForCheck('hasHighPriorityAdLoaded'),
-            waitForCheck('hasLowPriorityAdNotLoaded'),
```

Comment:
@shtukas I think the 2 function `hasHighPriorityAdLoaded` can `hasLowPriorityAdNotLoaded` can as well be removed as I think  they are used only by `IsOutbrainMerchandiseCompliant`.

Reply:
Done!